plugin/v2/plugin.go

```diff
@@ -250,12 +302,14 @@ func (p *Plugin) InitSpec(s specs.Spec, libRoot string) (*specs.Spec, error) {
 	if len(cwd) == 0 {
 		cwd = "/"
 	}
-	s.Process = specs.Process{
-		Terminal: false,
-		Args:     args,
-		Cwd:      cwd,
-		Env:      envs,
-	}
+	s.Process.Terminal = false
+	s.Process.Args = args
+	s.Process.Cwd = cwd
+	s.Process.Env = envs
+
+	// TODO: what about duplicates?
+	// TODO: Should not need CAP_ prefix in manifest?
+	s.Process.Capabilities = append(s.Process.Capabilities, p.PluginObj.Manifest.Capabilities...)
```

Comment:
Not your change, but we need some naming consistency here. "Capabilities" is overloaded:
1. plugin/store/store.go uses "Capability" to mean type of plugin. We should rename this to "pluginType" or simply "type".
2.  VolumeDriver.Capabilities is a endpoint for volume plugins. Nothing can be done here.
3. Manifest has a section for Linux capabilities (referenced here)